frontend/components/ButtonWithOptions/ButtonWithOptions.jsx

```diff
@@ -112,13 +118,13 @@ export default class ButtonWithOptions extends Component {
           onClick={this.toggleOptions.bind(this)}
         >
           <IconArrow
-            direction={this.state.optionsExpanded ? 'down' : 'up'}
+            direction={optionsExpanded ? 'down' : 'up'}
```

Comment:
You mean changing the state variable name from `optionsExpanded` to `open`? Just making sure, as I haven't touched that.